test/metabase/test/data/impl/get_or_create.clj

```diff
@@ -128,7 +128,12 @@
         (u/profile (format "%s %s Database %s (reference H2 duration: %s)"
                            (if full-sync? "Sync" "QUICK sync") driver database-name reference-duration)
           ;; only do "quick sync" for non `test-data` datasets, because it can take literally MINUTES on CI.
-          (binding [sync-util/*log-exceptions-and-continue?* false]
+          ;;
+          ;; MEGA SUPER HACK !!! I'm experimenting with this so Redshift tests stop being so flaky on CI! It seems like
+          ;; if we ever delete a table sometimes Redshift still thinks it's there for a bit and sync can fail because
+          ;; that it tries to sync a Table that is gone! So enable normal resilient sync behavior for Redshift tests to
+          ;; fix the flakes. If this fixes things I'll try to come up with a more robust solution. -- Cam 2024-07-19
+          (binding [sync-util/*log-exceptions-and-continue?* (not= driver :redshift)]
```

Comment:
Isn't this enabling robust behavior for drivers other than redshift?

Reply:
oops